generation/new_client/new-client.py

```diff
@@ -213,7 +213,14 @@ def generate(
 
     # Initialize workdir
     workdir = Path(f"{sys.path[0]}/../../java-{output_name}").resolve()
-    subprocess.run(["rm", "-fr", workdir])
+    if os.path.isdir(workdir):
+      sys.exit(
+          "Couldn't create the module because "
```

Comment:
```suggestion
          "ERROR: Couldn't create the module because "
```

Adding `ERROR:` for bit more visibility